qt/InteropHelper.cc

```diff
@@ -12,25 +12,19 @@
 
 bool InteropHelper::isConnected() const
 {
-#ifdef ENABLE_DBUS_INTEROP
+#if defined(ENABLE_DBUS_INTEROP)
 
-    if (myDbusClient.isConnected())
-    {
-        return true;
-    }
+    return myDbusClient.isConnected();
 
-#endif
+#elif defined(ENABLE_COM_INTEROP)
```

Comment:
DBus and COM are not mutually exclusive and Windows builds have both enabled, falling back from one to another.

Reply:
Nice catch, thank you for that! Fixed 6daf368dd76965d6d495d62444aaa6db48737d71